### ISSUE TITLE
[Crash]: Game crashes on world creation

### ISSUE THREAD

**Issue by NOTdenNes** [Fuzss/diagonal-blocks#55]
### Mod Loader (Required)

Forge

### Minecraft Version (Required)

1.20.1

### Mod Version (Required)

8.1.2

### Notes (Required)

_Assumed incompability with Domum Ornamentum: [https://www.curseforge.com/minecraft/mc-mods/domum-ornamentum](https://www.curseforge.com/minecraft/mc-mods/domum-ornamentum)_

Other mods installed:

- Diagonal Fences/Windows (ver. 8.1.2)
- Puzzles Lib (ver. 8.1.8)
- Domum Ornamentum (ver. 1.0.147-BETA)

How to replicate:

1. Start Game
2. Click 'Singleplayer'
3. **Crash**

### Crash Report (Required)

https://gist.github.com/NOTdenNes/83ed61a0ca7afb9ebc10fb61bdd2ae70

### latest.log (Optional)

https://gist.github.com/NOTdenNes/4736dc7a038f2e3b85debf9d60362560

**Comment by Fuzss** [comment]
Does it work without Domum Ornamentum?
And does it always crash or just sometimes?

**Comment by NOTdenNes** [comment]
Yes, it does work without Domum Ornamentum, and DO does work without Diagonal Windows/Fences.
Also, the game crashes every single time when having both/all three mods active.
